### DISCUSSION TITLE
Browser: Connection "not secure" (ManagementCA.pem)

### DISCUSSION THREAD

**Discussion by schefa11** [Keyfactor/ejbca-ce#338] [Q&A] [answered]
Hi,

I successfully issued client certificates for a webserver (nginx) to a raspberry via SCEP.
I imported the ManagementCA.pem to my Firefox Browser and the Chrome Browser, but they still show a warning.

The common name of the client certificate is the IP-address of the raspberry.

Chrome: NET::ERR_CERT_COMMON_NAME_INVALID
![grafik](https://github.com/Keyfactor/ejbca-ce/assets/37020087/a688a44f-461c-4cc0-82cc-3d2ccd3e4c89)
(My certificates are only valid for 30min for testing, but I don't think that is the issue)

Firefox: SEC_ERROR_INADEQUATE_CERT_TYPE
On Firefox I cannot even access the webserver, until I delete the certificate from the browser.
![grafik](https://github.com/Keyfactor/ejbca-ce/assets/37020087/9b9fab8c-c9b5-4e02-914d-e6303d79798b)

Recommendations are welcome. Thanks in advance.

**Comment by primetomas** [comment]
The browser error actually tells it. The server IP (or DNSName) should go into a Subject Alternative Name, not in the Common Name. The usage of common name for the sever adress was deprecated a long time ago. In EJBCA add alternative names (IP and/or dns) in the end entity profile and issue certs with the IP in the alternative name.

**Reply by schefa11** [reply]
Thanks for the fast response.

I now get the message "NET::ERR_CERT_INVALID".
![image](https://github.com/Keyfactor/ejbca-ce/assets/37020087/cbc9c36e-d1d6-4c98-b17b-5781277843aa)

But maybe I have a misconception?!
As mentioned before I created certificates based on the TLS Client Profile. But since the certificates are meant as webserver certificates for the nginx it probably needs to be based on the TLS Server Profile?

**Reply by primetomas** [reply, marked as answer]
Correct, for servers there are specific extension values needed, you need to use a server profile.

**Reply by schefa11** [reply]
Thank you very much. Now it is working as intended
...again :)
